### ISSUE TITLE
Not working after latest update

### ISSUE THREAD

**Issue by gamingneeds** [ExcuseMi/rocketleaguelivestats#3]
New RL Patch seems to have killed it.

**Comment by ExcuseMi** [comment]
I'll check on this later, sorry for the delay.
You sure the API is still working?

**Comment by gamingneeds** [comment]
Yeah. It looks like the data isn't  in the log anymore. :/

**Comment by gamingneeds** [comment]
Were you able to check this out?

**Comment by gamingneeds** [comment]
You still around? Seems you disappeared. :(

**Comment by ExcuseMi** [comment]
Yeah, I did check. It seems like the data needed isn't in the logs anymore.

**Comment by niuo** [comment]
Hello, your tool seems to be very good.
Unfortunately I am on the last version of RL and your tool doesn't work.
I don't understand if this current issue can be fixed with code or Psyonix blocked the API ?
EDIT: Ok I checked the code and I understand now. The Launch.log doesn't contain the players ID anymore ;(
